schemas/api.yaml

```diff
@@ -103,3 +155,17 @@ components:
       properties:
         name:
           type: string
+    DetailedUpstreamBranch:
+      type: object
+      required:
+        - name
+        - color
+        - exists
+        - behindCount
+        - hasConflict
+      properties:
+        name: { type: string }
```

Comment:
NIT, should we try to stick to a single format for field definitions? Sometimes it's a JSON looking format like we have here and in other places it's YAML like on line 95.

Reply:
Yeah, it'd be nice to have some sort of yaml linter here, agreed.

I like using the inline objects like this for property lists; it keeps each property on one line like I'd expect for `string name`. Unfortunately, I'm definitely not consistent on this, so a tool that at least makes it consistent would help.

I'll investigate using Prettier on these files for a future branch.

Reply:
Prettier is actually already enabled here 😞 Seems like it's not opinionated enough. I'll look further into other tools.